formbuilder/styles.css

```diff
@@ -153,3 +153,7 @@ textarea.json-viewer {
 .builder-inner-actions {
   margin-bottom: 30px;
 }
+
+.checkbox label strong{
```

Comment:
currently, to set spacing between checkbox and label, bootstrap use:

```
.radio label, .radio-inline label, 
.checkbox label, .checkbox-inline label {
    padding-left: 25px
}
```
```
.radio input[type="radio"], 
.radio-inline input[type="radio"], 
.checkbox input[type="radio"], 
.checkbox-inline input[type="radio"], 
.radio input[type="checkbox"], 
.radio-inline input[type="checkbox"], 
.checkbox input[type="checkbox"], 
.checkbox-inline input[type="checkbox"]  {
    margin-left: -25px
}

```

What you want here is to replace 25px by 28px. 
The best way to do it would be to use bootstrap's variables file.
The quicker way is to repeat the selectors in our styles.css